.github/workflows/release-next.yaml

```diff
@@ -47,24 +41,29 @@ jobs:
       - setup
     runs-on: ubuntu-latest
     steps: 
-      - name: Cache bin path
-        id: cache
-        uses: actions/cache@v2
-        with:
-          path: ~/bin
-          key: ${{ runner.os }}-${{ needs.setup.outputs.HELM_VERSION }}-${{ needs.setup.outputs.YQ_VERSION }}
       - name: Checkout
         uses: actions/checkout@v2
-      - run: scripts/install-helm.sh $HELM_VERSION
-        if: steps.cache.outputs.cache-hit != 'true'
-      - run: scripts/install-yq.sh $YQ_VERSION
-        if: steps.cache.outputs.cache-hit != 'true'
+      - run: scripts/install-helm.sh ${{ needs.setup.outputs.HELM_VERSION }}
+      - run: scripts/install-yq.sh ${{ needs.setup.outputs.YQ_VERSION }}
       - name: Update PATH
         run: |
           echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
       - name: Update chart dependencies
         run: |
           scripts/update-helm-deps.sh
+      - name: Compute release chart version
+        id: compute_next_version
+        run: |
+          DATE_TIME=$(date '+%Y%m%d%H%M%S')
+          NEXT_VERSION=$(echo $(yq e '.version' charts/k8ssandra/Chart.yaml) | sed "s/-.*/-${DATE_TIME}-${GITHUB_SHA::8}/")
```

Comment:
@adejanovski and I talked about version number format previously and my biggest concern was that the format we use plays nicely with Semver. We're good to go here 👍